README.md

```diff
@@ -10,24 +10,65 @@ This repository contains OpenEBS Helm charts and other example artifacts like op
 
 ## OpenEBS Helm Chart
 
-The helm chart is located under [./charts/](./charts/) directory. 
+The helm chart is located under [./charts/openebs/](./charts/openebs/) directory. 
 
-When new changes to helm chart are pushed to master branch, the changes are picked up by [Helm Chart releaser](https://github.com/helm/chart-releaser-action) GitHub Action. The chart releaser will: 
-- Upload the new version of the charts to the [GitHub releases](https://github.com/openebs/charts/releases).
-- Update the helm repo index file and push to the [GitHub Pages branch](https://github.com/openebs/charts/tree/gh-pages).
+OpenEBS helm chart is an umbrella chart that pulls together engine specific charts. The engine charts are included as dependencies in [Chart.yaml](charts/openebs/Chart.yaml).
 
-## Mayastor Helm Chart
+OpenEBS helm chart will include templates for the common tools or components that are used by multiple engines like:
+- Node Disk Manager related components
+- Dynamic Local Provisioner related components
+- Security Policies like RBAC, PSP, Kyverno 
+- cStor and Jiva out-of-tree provisioners that will be replaced by respective engine chart listed below
+
+Engine charts included as dependencies are:
+- [cStor](https://github.com/openebs/cstor-operators/tree/master/deploy/helm/charts)
+- [Jiva](https://github.com/openebs/jiva-operator/tree/master/deploy/helm/charts)
+- [ZFS Local PV](https://github.com/openebs/zfs-localpv/tree/master/deploy/helm/charts)
+- [LVM Local PV](https://github.com/openebs/lvm-localpv/tree/master/deploy/helm/charts)
```

Comment:
Shall we add dynamic localpv chart as well

Reply:
Not required as we want to install them by default.